package-lock.json

```diff
@@ -1701,7 +1701,7 @@
         "get-value": "^2.0.6",
         "has-value": "^1.0.0",
         "isobject": "^3.0.1",
-        "set-value": "^2.0.1",
+        "set-value": "^2.0.0",
```

Comment:
>"set-value": "^2.0.0", [](start = 8, length = 22)

I am surprised we would ever go back a version, and a previous version of "set-value" have vulnerability we recently had to move up to mitigate, so please make sure this was needed to revert to a previous version.

Reply:
There is something odd with npm, audit command or otherwise. It picked these values based on the new dependency. Why it downgraded existing ones, brought in I suppose by just pulling a top level dependency, beats me.